File: 백준/Bronze/30204. 병영외 급식/병영외 급식.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int X = sc.nextInt();
        int[] Ps = new int[N];
        int sum = 0;
        for (int i = 0; i < N; i++) {
            Ps[i] = sc.nextInt();
            sum += Ps[i];
        }
        if (sum % X == 0) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
        sc.close();
    }
}

File: 백준/Bronze/30204. 병영외 급식/병영외 급식.c
#include <stdio.h>

int main() {
    int N, X;
    scanf("%d %d", &N, &X);
    int Ps[N];
    int sum = 0;
    for (int i = 0; i < N; i++) {
        scanf("%d", &Ps[i]);
        sum += Ps[i];
    }
    if (sum % X == 0) {
        printf("1\n");
    } else {
        printf("0\n");
    }
    return 0;
}